linux_os/guide/system/software/integrity/crypto/harden_openssl_crypto_policy/tests/incorrect_followed_by_correct.pass.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# platform = multi_platform_fedora,Red Hat Enterprise Linux 8
+# profiles = xccdf_org.ssgproject.content_profile_ospp
```

Comment:
Will you add the rule `harden_openssl_crypto_policy` to the ospp profile? At this moment, the rule isn't part of any profile.

This causes the test suite to shout.

```
[jcerny@thinkpad tests{pr/5925}]$ python3 test_suite.py rule --libvirt qemu:///system ssgts_rhel8 --datastream ../build/ssg-rhel8-ds.xml harden_openssl_crypto_policy
Setting console output to log level INFO
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/jcerny/work/git/scap-security-guide/tests/logs/rule-custom-2020-07-14-1354/test_suite.log
INFO - xccdf_org.ssgproject.content_rule_harden_openssl_crypto_policy
ERROR - Script incorrect_followed_by_correct.pass.sh using profile xccdf_org.ssgproject.content_profile_ospp found issue:
ERROR - Rule xccdf_org.ssgproject.content_rule_harden_openssl_crypto_policy has not been evaluated! Wrong profile selected in test scenario?
ERROR - The initial scan failed for rule 'xccdf_org.ssgproject.content_rule_harden_openssl_crypto_policy'.
ERROR - Script empty_policy.fail.sh using profile xccdf_org.ssgproject.content_profile_ospp found issue:
ERROR - Rule xccdf_org.ssgproject.content_rule_harden_openssl_crypto_policy has not been evaluated! Wrong profile selected in test scenario?
ERROR - The initial scan failed for rule 'xccdf_org.ssgproject.content_rule_harden_openssl_crypto_policy'.
ERROR - Script correct_followed_by_incorrect.fail.sh using profile xccdf_org.ssgproject.content_profile_ospp found issue:
ERROR - Rule xccdf_org.ssgproject.content_rule_harden_openssl_crypto_policy has not been evaluated! Wrong profile selected in test scenario?
ERROR - The initial scan failed for rule 'xccdf_org.ssgproject.content_rule_harden_openssl_crypto_policy'.
ERROR - Script correct.pass.sh using profile xccdf_org.ssgproject.content_profile_ospp found issue:
ERROR - Rule xccdf_org.ssgproject.content_rule_harden_openssl_crypto_policy has not been evaluated! Wrong profile selected in test scenario?
ERROR - The initial scan failed for rule 'xccdf_org.ssgproject.content_rule_harden_openssl_crypto_policy'.
ERROR - Script missing_file.fail.sh using profile xccdf_org.ssgproject.content_profile_ospp found issue:
ERROR - Rule xccdf_org.ssgproject.content_rule_harden_openssl_crypto_policy has not been evaluated! Wrong profile selected in test scenario?
ERROR - The initial scan failed for rule 'xccdf_org.ssgproject.content_rule_harden_openssl_crypto_policy'.
ERROR - Script correct_commented.fail.sh using profile xccdf_org.ssgproject.content_profile_ospp found issue:
ERROR - Rule xccdf_org.ssgproject.content_rule_harden_openssl_crypto_policy has not been evaluated! Wrong profile selected in test scenario?
ERROR - The initial scan failed for rule 'xccdf_org.ssgproject.content_rule_harden_openssl_crypto_policy'.
ERROR - Script incorrect_policy.fail.sh using profile xccdf_org.ssgproject.content_profile_ospp found issue:
ERROR - Rule xccdf_org.ssgproject.content_rule_harden_openssl_crypto_policy has not been evaluated! Wrong profile selected in test scenario?
ERROR - The initial scan failed for rule 'xccdf_org.ssgproject.content_rule_harden_openssl_crypto_policy'.
```

Also the profiles comment in test scenarios  header isn't necessary when the rule isn't parametrized by an XCCDF value.

Reply:
No, the rule is here to fullfill specific requirement and it won't be placed into upstream profile.

Reply:
@vojtapolasek OK, then why do you put `profiles` to the test scenario header?